public/API/API_GetUserWantToPlayList.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
```

Comment:
nit: In this doc block, make sure all documentation indentation is consistent. There is too much indentation in the last two items:

```
 *    string     ConsoleName                name of the console associated to the game
 *    string     ImageIcon                  site-relative path to the game's icon image
 *    int        TotalPoints                     total points able to be earned
 *    int        NumPossibleAchievements            total number of achievements to be unlocked
```

Reply:
`NumPossibleAchievements` should also be renamed. Please do a one-over of the doc block to ensure it's consistent with the latest implementation details.

Reply:
Adjusted indents and made sure docs look right